test/execution/test_translation.py

```diff
@@ -40,6 +42,64 @@ def foo(x, state):
     assert (y == th.tensor([3.0, 5])).all()
 
 
+def test_func_plan_can_be_translated_to_tfjs(hook, workers):
+    Plan._build_translators = []
+
+    @sy.func2plan(args_shape=[(3, 3)])
+    def plan(x):
+        x = x * 2
+        x = x.abs()
+        return x
+
+    orig_plan = plan.copy()
+
+    plan_js = plan.copy()
+    plan_js.add_translation(PlanTranslatorTfjs)
+    plan_js.base_framework = TranslationTarget.TENSORFLOW_JS.value
+    assert plan_js.role.actions[0].name == "tf.mul"
+    assert len(plan_js.role.actions[0].args) == 2
+
+    # Test plan caching
+    plan_js2 = plan_js.copy()
+    plan_js2.add_translation(PlanTranslatorTfjs)
+    plan_js2.base_framework = TranslationTarget.TENSORFLOW_JS.value
+    assert plan_js2.role.actions[0].name == "tf.mul"
+    assert len(plan_js2.role.actions[0].args) == 2
+
+    # check that translation can be done after serde
+    serde_plan = deserialize(serialize(orig_plan))
+    serde_plan.add_translation(PlanTranslatorTfjs)
+    serde_plan.base_framework = TranslationTarget.TENSORFLOW_JS.value
+    assert serde_plan.role.actions[0].name == "tf.mul"
+    assert len(serde_plan.role.actions[0].args) == 2
+
+    # check that translation is not lost after serde
+    serde_plan_full = deserialize(serialize(plan_js))
+    assert serde_plan_full.role.actions[0].name == "tf.mul"
+    assert len(serde_plan_full.role.actions[0].args) == 2
+
+
+@pytest.mark.skip(reason="Missing translation for torch.nn.functional.linear")
```

Comment:
Are we not allowed to have skipped tests? It would be great to keep this in here until threepio support comes about (which is soon)

Reply:
Hmm, I agree, we should be able to skip tests. Not sure why this counts toward coverage. Quick fix would probably be to keep the test here but comment it out, and we can uncomment it if we figure out a way to make skipped tests not count toward coverage requirements.